src/cascadia/TerminalSettingsEditor/Launch.xaml

```diff
@@ -201,6 +201,17 @@
                                         Value="{x:Bind State.Settings.GlobalSettings.InitialRows, Mode=TwoWay}" />
                     </Grid>
                 </local:SettingContainer>
+
+                <!--  ConsoleHost startup mode  -->
+                <local:SettingContainer x:Name="Globals_ConsoleHostStartupMode"

```

Comment:
(definitely a follow-up, but worth mentioning) I think this setting should be disabled if the default terminal isn't set to the current app. That kind of logic would require an MVVM pattern for the settings UI though, and unfortunately that's not done yet. I think it'd be valuable to do that as a follow-up though, so maybe we should file this as a separate issue?